x-pack/plugins/lens/public/editor_frame_service/editor_frame/suggestion_panel.scss

```diff
@@ -19,6 +15,15 @@
   padding-bottom: $euiSizeXS;
 }
 
+.lnsSuggestionPanel__accordion {
+  // this makes the accordion content scrollable
+  display: inline-grid;
+}
+
+.lnsSuggestionPanel--hideSuggestions {
+  padding-bottom: $euiSizeS;
+}
```

Comment:
The reason I didn't do it this way, is because I saw this comment
```
.lnsFrameLayout__pageBody {
  @include euiScrollBar;
  min-width: $lnsPanelMinWidth + $euiSizeXL;
  overflow: hidden auto;
  display: flex;
  flex-direction: column;
  flex: 1 1 100%;
  // Leave out bottom padding so the suggestions scrollbar stays flush to window edge
  // Leave out left padding so the left sidebar's focus states are visible outside of content bounds
  // This also means needing to add same amount of margin to page content and suggestion items**
  padding: $euiSize $euiSize 0;
  position: relative;
  z-index: $lnsZLevel1;

  &:first-child {
    padding-left: $euiSize;
  }

  &.lnsFrameLayout__pageBody-isFullscreen {
    background: $euiColorEmptyShade;
    flex: 1;
    padding: 0;
  }
}
```

But I removed the condition with this way:
- removed the `lnsSuggestionPanel--hideSuggestions `
- removed the padding from `lnsSuggestionPanel__suggestions`
- added the padding on the container `lnsSuggestionPanel`
Wdyt about that?

Reply:
That works. Thanks for looking into it!